### PR TITLE
AI Oceans: initial skeleton for code.org/hourofcode/overview 

### PR DESCRIPTION
**Pull request by Erin007**
Users viewing the page in English (and launch_ai DCDO flag set for now): 
[eng-ai-launch-hoc-overview.pdf](https://github.com/code-dot-org/code-dot-org/files/3876968/eng-ai-launch-hoc-overview.pdf)

Users viewing the page in a language other than English  (no change from current):
non-en-non-ai

Still waiting on and will include in a follow-up:
- [ ]  AI assets for the top promo box 
- [ ] CS For Good assets from the the teal HOC banner mid-page
- [ ] Link to the curriculum guide

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/hourofcode/overview.haml [hunk 1]
```diff
@@ -16,6 +16,8 @@ social:
   @header["social"]["og:description"] = hoc_s(:social_hoc2018_global_movement)
   @header["social"]["og:image"] = 'https://' + request.host + '/images/social-media/code-2018-creativity.png'
 
+  launch_ai = DCDO.get('launch_ai', nil)
```
Comment (maddiedierker) [inline pegasus/sites.v3/code.org/public/hourofcode/overview.haml:19]: this needs a `-` for HAML to read this as a variable assignment, right?

```suggestion
  - launch_ai = DCDO.get('launch_ai', nil)
```

Comment (Erin007) [reply pegasus/sites.v3/code.org/public/hourofcode/overview.haml:19]: It's still in the `:ruby` block from line 10 (which you can't see in this diff 🙂)

Comment (maddiedierker) [reply pegasus/sites.v3/code.org/public/hourofcode/overview.haml:19]: woops, you're right!

**Review by maddiedierker — APPROVED** [review]
💃 🐟

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 2]
```diff
@@ -22,6 +22,9 @@
   social_hoc2019_coming_dates: 'The Hour of Code is coming December 9-15, 2019. Computer science is foundational for every student to learn.'
   social_hoc2019_dance: 'Featuring Katy Perry, Shawn Mendes, Panic! At The Disco, Lil Nas X, Jonas Brothers, Nicki Minaj, and 34 more!'
 
+  hoc_overview_oceans_tutorial_header: 'AI Oceans'
+  hoc_overview_oceans_tutorial_desc: "Teach \"A.I.\" to sort between fish and trash while exploring broader artificial intelligence concepts."
```
Comment (breville) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml:26]: fwiw, you can also wrap in single quotes to avoid the escaping.

**Review by breville — APPROVED** [review]

**Pull request merged**